### PR TITLE
Fix txt

### PR DESCRIPTION
**Pull request by JLHwung**
Companion PR: #2

### CODE REVIEW

File: KR2k0042_003.txt [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+#+PROPERTY: JUAN 卷三
+<pb:KR2k0042_WYG_003-1a>¶
+欽定四庫全書¶
+　江南通志卷三¶
+　　輿地志¶
+　　　圖¶
+　　　黃河入江南界圖¶
+　　　淮水入江南界圖¶
+　　　江南界漕河圖¶
+　　　高家𤦵圖¶
```
Comment (JLHwung) [inline KR2k0042_003.txt:15]: Proofread link: http://img.kanripo.org/general/skqs/wyg/WYG0507/WYG0507-0181a.png

𤦵 may be typo for 堰. Here we transcribe faithfully.

File: KR2k0042_003.txt [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+<pb:KR2k0042_WYG_003-1b>¶
+　　　江防圖¶
+　　　海防圖¶
+　　　太湖水利圖¶
+¶
+¶
+¶
+¶
```
Comment (JLHwung) [inline KR2k0042_003.txt:23]: Proofread link: Proofread link: http://img.kanripo.org/general/skqs/wyg/WYG0507/WYG0507-0181b.png

File: KR2k0042_003.txt [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<pb:KR2k0042_WYG_003-35b>¶
+¶
+¶
+¶
+¶
+¶
+¶
+¶
+　江南通志卷三¶
```
Comment (JLHwung) [inline KR2k0042_003.txt:100]: Proofread link: http://img.kanripo.org/general/skqs/wyg/WYG0507/WYG0507-0198b.png

**Comment by cwittern** [comment]
Great work, thanks.

**Pull request merged**